src/p2p/net_node.inl

```diff
@@ -158,17 +158,18 @@ namespace nodetool
     m_config.m_net_config.connection_timeout = P2P_DEFAULT_CONNECTION_TIMEOUT;
     m_config.m_net_config.ping_connection_timeout = P2P_DEFAULT_PING_CONNECTION_TIMEOUT;
     m_config.m_net_config.send_peerlist_sz = P2P_DEFAULT_PEERS_IN_HANDSHAKE;
+    m_config.m_support_flags = P2P_SUPPORT_FLAG_FLUFFY_BLOCKS; // | OTHER_FLAGS
```

Comment:
Subjective, but I'd put a P2P_SUPPORT_FLAGS in cryptonote_config.h, initialized to that one flag, Makes it easier to find when adding another flag.